src/Umbraco.Infrastructure/Composing/CompositionExtensions/Services.cs

```diff
@@ -100,9 +100,9 @@ private static LocalizedTextServiceFileSources SourcesFactory(IFactory container
         {
             var hostingEnvironment = container.GetInstance<IHostingEnvironment>();
             var globalSettings = container.GetInstance<IGlobalSettings>();
-            var mainLangFolder = new DirectoryInfo(hostingEnvironment.MapPathContentRoot(globalSettings.UmbracoPath + "/config/lang/"));
+            var mainLangFolder = new DirectoryInfo(hostingEnvironment.MapPathContentRoot(Path.Combine(globalSettings.UmbracoPath , "config","lang")));
```

Comment:
Although this 'works' it is incorrect. Path.Combine on windows will use backslashes which the browser and our own methods will magically fixup but we shouldn't rely on that. We cannot use Path.Combine for combining web paths. We need our own utility for combining web paths like `WebPath.Combine`, apparently you can use a trick using Uri https://stackoverflow.com/questions/372865/path-combine-for-urls